src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_data_connection/manage_direct_query_data_connections_table.tsx

```diff
@@ -151,190 +221,88 @@ export const ManageDirectQueryDataConnectionsTable: React.FC<ManageDirectQueryDa
-      render: (value, record: DataConnection) => (
-        <EuiFlexGroup>
-          <EuiFlexItem grow={false}>{icon(record)}</EuiFlexItem>
-          <EuiFlexItem grow={false}>
-            <EuiLink
-              data-test-subj={`${record.name}DataConnectionsLink`}
-              onClick={() =>
-                history.push(`/manage/${record.name}?dataSourceMDSId=${selectedDataSourceId ?? ''}`)
-              }
-            >
-              {truncate(record.name, 100)}
-            </EuiLink>
-          </EuiFlexItem>
-        </EuiFlexGroup>
-      ),
     },
     {
-      field: 'status',
-      name: 'Status',
-      sortable: true,
+      width: '35%',
+      field: 'description',
+      name: 'Description',
       truncateText: true,
-      render: (value, record: DataConnection) =>
-        record.dsStatus === 'ACTIVE' ? (
-          <EuiHealth color="success">Active</EuiHealth>
-        ) : (
-          <EuiHealth color="subdued">Inactive</EuiHealth>
-        ),
+      mobileOptions: {
+        show: false,
+      },
     },
     {
-      field: 'actions',
-      name: 'Actions',
-      actions,
+      field: 'relatedConnections',
+      name: 'Related connections',
+      align: 'right',
+      truncateText: true,
+      render: (relatedConnections: DataSourceTableItem[]) => relatedConnections?.length,
     },
-  ] as Array<EuiTableFieldDataColumnType<unknown>>;
+  ] as Array<EuiTableFieldDataColumnType<DataSourceTableItem>>;
 
-  const customSearchBar = (
-    <EuiFlexGroup gutterSize="s" justifyContent="flexEnd">
-      {featureFlagStatus && (
-        <EuiFlexItem grow={false} style={{ width: '30%' }}>
-          <DataSourceSelector
-            savedObjectsClient={savedObjects.client}
-            notifications={notifications.toasts}
-            onSelectedDataSource={handleSelectedDataSourceChange}
-            fullWidth={true}
-            uiSettings={uiSettings}
-            disabled={false}
-            compressed={true}
-            hideLocalCluster={getHideLocalCluster().enabled}
-            isClearable={false}
-          />
-        </EuiFlexItem>
-      )}
-      <EuiFlexItem grow={true}>
-        <EuiCompressedFieldSearch
-          placeholder="Search..."
-          value={searchText}
-          onChange={(e) => setSearchText(e.target.value)}
-          isClearable
-          fullWidth={true}
-        />
-      </EuiFlexItem>
-    </EuiFlexGroup>
-  );
-
-  const entries = data.filter((dataconnection) =>
-    dataconnection.name.toLowerCase().includes(searchText.toLowerCase())
-  );
+  const customSearchBar: EuiSearchBarProps = {
+    toolsLeft: renderToolsLeft(),
+    box: {
+      incremental: true,
+    },
+    compressed: true,
+    filters: [
+      {
+        type: 'field_value_selection',
+        field: 'type',
+        name: 'Type',
+        multiSelect: 'or',
+        options: Array.from(new Set(data.map(({ type }) => type).filter(Boolean))).map((type) => ({
+          value: type!,
+          name: type!,
```

Comment:
is the `name` displayed on the UI? If yes, we should find a way to use i18n for it.